actions/src/server_query_actions.py

```diff
@@ -0,0 +1,47 @@
+from typing import Callable, List
+from st2common.runners.base_action import Action
+
+from enums.query.props.server_properties import ServerProperties
+from enums.query.query_output_types import QueryOutputTypes
+from enums.cloud_domains import CloudDomains
+
+from openstack_query.managers.server_manager import ServerManager
+from structs.query.query_output_details import QueryOutputDetails
+
+
+class ServerQueryActions(Action):
+    """
+    Stackstorm Action class that dynamically dispatches actions related to Server Queries to the corresponding
+    method in the ServerManager class
+    Actions that will be handled by this class follow the format server.search.*
+    """
+
+    def __init__(self, *args, **kwargs):
+        self._server_manager = ServerManager(cloud_account=CloudDomains)
```

Comment:
Why do we pass in the Enum here? If this is a hangover from DI + Testing we should just make `CloudDomains` to default type in `init`